cmd/kubeadm/app/util/error.go

```diff
@@ -63,6 +65,9 @@ func checkErr(prefix string, err error, handleErr func(string, int)) {
 		return
 	case *preflight.Error:
 		handleErr(err.Error(), PreFlightExitCode)
+	case utilerrors.Aggregate:
```

Comment:
While this is the case now, it might not be in the future. Can we find a way to do it better?